Makefile

```diff
@@ -17,8 +17,8 @@ SRC_FILES=$(shell find . -name '*.go' | grep -v '_test.go' | grep -v '/tests/')
 TEST_FILES=$(wildcard *_test.go)
 TELEMETRY_TEST_DIR = build/tests/gnmi_server
 TELEMETRY_TEST_BIN = $(TELEMETRY_TEST_DIR)/server.test
-ifeq ($(TELEMETRY_WRITABLE),y)
-BLD_FLAGS := -tags readwrite
+ifeq ($(TRANSLIB_ENABLED),y)
```

Comment:
>TRANSLIB_ENABLED [](http://example.com/codeflow?start=8&length=16)

Since you rename it, will you raise a buildimage PR? Mention the related PRs in this PR description.